functions/generate_pe_conf.pp

```diff
@@ -16,23 +15,26 @@ function peadm::generate_pe_conf (
   # Define the configuration settings that will be placed in pe.conf by
   # default. These can be overriden by user-supplied values in the $settings
   # hash.
+  #
+  # At this time, there are no defaults which need to be modified from the
+  # out-of-box defaults.
   $defaults = {
```

Comment:
If no one is using this, and there are no use-cases in the near future ... should we remove the whole `$defaults` hash definition?

Reply:
I thought about it, but wondered if there was value in keeping the structure there as scaffolding to save us a little time in the event a future change forces us back into setting some defaults. It doesn't feel like it costs anything to keep it.